examples/bri-3/src/bri/identity/bpiSubjects/capabilities/updateBpiSubject/updateBpiSubjectCommand.handler.ts

```diff
@@ -14,11 +14,26 @@ export class UpdateBpiSubjectCommandHandler
   ) {}
 
   async execute(command: UpdateBpiSubjectCommand) {
-    const bpiSubjectToUpdate: BpiSubject =
+    const bpiSubject: BpiSubject =
       await this.agent.fetchUpdateCandidateAndThrowIfUpdateValidationFails(
         command.id,
       );
-    Object.assign(bpiSubjectToUpdate, command);
+
+    const bpiSubjectToUpdate = new BpiSubject(
```

Comment:
This mapping already inside of fetchUpdateCandidateAndThrowIfUpdateValidationFails or am i missing something?

Reply:
Oh right, i guess I changed this already. Let me check please if I committed this

Reply:
Oh, I just checked. 
I think you were checking the older commit. please check this commit: 

https://github.com/eea-oasis/baseline/pull/528/commits/6ebd72657ecef3e49135ceffebd15ea8e9b31cd2